{
  "Header": {
    "Original": "CVE-2021-45111 (Improper access control in Odoo Community 15.0 and earlier and Odoo En ...)",
    "Line": 110346,
    "ID": "CVE-2021-45111",
    "Description": "(Improper access control in Odoo Community 15.0 and earlier and Odoo En ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5399-1}",
      "Line": 110347,
      "Type": "xref",
      "Bugs": [
        "DSA-5399-1"
      ]
    },
    {
      "Original": "- odoo \u003cunfixed\u003e (bug #1035953)",
      "Line": 110348,
      "Type": "package",
      "Package": "odoo",
      "Kind": "unfixed",
      "BugNo": 1035953
    },
    {
      "Original": "NOTE: https://github.com/odoo/odoo/issues/107683",
      "Line": 110349,
      "Type": "NOTE",
      "Description": "https://github.com/odoo/odoo/issues/107683"
    },
    {
      "Original": "NOTE: 14.0 patch at https://github.com/odoo/odoo/commit/d326153e016f93c22f40ad8fb146bb4108bb94dc",
      "Line": 110350,
      "Type": "NOTE",
      "Description": "14.0 patch at https://github.com/odoo/odoo/commit/d326153e016f93c22f40ad8fb146bb4108bb94dc"
    }
  ]
}